submissions/Ant-C-tech/html-movie-seat-booking/index.html

```diff
@@ -0,0 +1,477 @@
+          <p class="movieDate__title"><time datetime="2021-03-02">Tue, 2 March</time></p>
+          <div class="flexRow">
+            <a href="#" class="btn-prevDay">&#9668;</a>
+              <ol class="calendar">
+                <li class="calendar__listItem">
+                  <a href="#" class="calendar__listItemLink calendar__listItemLink">
+                    <span class="calendar__supText calendar__supText-today">Today</span><br>
+                    <time datetime="2021-03-01">1</time>
+                  </a>
+                </li>
+                <li class="calendar__listItem">
+                  <a href="#" class="calendar__listItemLink calendar__listItemLink-active">
+                    <span class="calendar__supText calendar__supText-active">Tue</span><br>
+                    <time datetime="2021-03-02">2</time>
+                  </a>
+                </li>
+                <li class="calendar__listItem">
+                  <a href="#" class="calendar__listItemLink">
+                    <span class="calendar__supText">Wed</span><br>
+                    <time datetime="2021-03-03">3</time>
+                  </a>
+                </li>
+                <li class="calendar__listItem">
+                  <a href="#" class="calendar__listItemLink">
+                    <span class="calendar__supText">Thu</span><br>
+                    <time datetime="2021-03-04">4</time>
+                  </a>
+                </li>
+                <li class="calendar__listItem">
+                  <a href="#" class="calendar__listItemLink calendar__listItemLink-disabled">
+                    <span class="calendar__supText calendar__supText-disabled">Fri</span><br>
+                    <time datetime="2021-03-05">5</time>
+                  </a>
+                </li>
+              </ol>
+            <a href="#" class="btn-nextDay">	&#9658;</a>
+          </div>
+        </section>
+
+        <section class="movieTime">
+          <p class="movieTime__title">Time</p>
+          <ul class="movieTime__list">
+            <li class="movieTime__listItem  movieTime__listItem-active">
+              <a class="movieTime__listItemLink  movieTime__listItemLink-active" href="#">
+                <time datetime="14:00">14:00</time>
+              </a>
+            </li>
+            <li class="movieTime__listItem">
+              <a class="movieTime__listItemLink" href="#">
+                <time datetime="17:00">17:00</time>
+              </a>
+            </li>
+            <li class="movieTime__listItem">
+              <a class="movieTime__listItemLink" href="#">
+                <time datetime="20:00">20:00</time>
+              </a>
+            </li>
+            <li class="movieTime__listItem  movieTime__listItem-disabled">
+              <a class="movieTime__listItemLink movieTime__listItemLink-disabled" href="#">
+                <time datetime="23:00">23:00</time>
+              </a>
+            </li>
```

Comment:
As for me, it looks like a group of radio buttons: group radio buttons allow you to choose only one item from the group. It's native behaviour in browser no need to add js for it.